7_deploy_osh/roles/suse-osh-setup-workers/tasks/subvolumes-setup.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Add subvolume setup script
+  script: caasp_cp_node_set_subvolumes.sh
+  register: _subvolume_setup
+  args:
+    creates: /var/lib/nova
+
+- name: Reboot machine
+  reboot:
```

Comment:
Reboot module is added in 2.7.x version which is a very recent ansible version. We may want to add logic (possibly in pre-flight validations) to warn user if ansible version is less than required version. Most OS will not have that version available by default in their installation.